en/material-system/effect-syntax.md

```diff
@@ -212,18 +231,21 @@ Relevant details:
 * All instanced properties are input attributes of the vertex shader, so if some property is needed in fragment shader, you need to pass it as varyings;
 * Make sure the code works for all branches, regardless of the actual state of `USE_INSTANCING`;
 
-The instanced property value will be initialized to all zeros by default.<br>
+The instanced property value will be initialized to all zeros by default.
+
 Use the `setInstancedAttribute` on `ModelComponent` to assign new values:
+
 ```ts
 const comp = node.getComponent(ModelComponent);
 comp.setInstancedAttribute('a_instanced_color', [100, 150, 200, 255]); // should match the specified format
 ```
-> Note: The instanced properties will be reset to all zeros whenever creating a new PSO (the most common case is when assigning a new material).
+> **Note**: The instanced properties will be reset to all zeros whenever creating a new PSO (the most common case is when assigning a new material).
 
 ### WebGL 1 fallback Support
-The effect compiler provides fallback conversion from GLSL 300 ES to GLSL 100 automatically, for WebGL 1.0 only support GLSL 100 syntax. this should be transparent to developers for the most time.<br>
-Currently the automatic conversion only supports some basic usage, and if some post-100 features or extensions were used, (texelFetch, textureGrad, etc.)<br>
-you have to do your own porting using the language built-in \_\_VERSION__ macro:
+The effect compiler provides fallback conversion from GLSL 300 ES to GLSL 100 automatically, for WebGL 1.0 only support GLSL 100 syntax. this should be transparent to developers for the most time.
+
+Currently the automatic conversion only supports some basic usage, and if some post-100 features or extensions were used, (texelFetch, textureGrad, etc.) Developers have to do your own porting using the language built-in __VERSION__ macro:
```

Comment:
The macro name is `__VERSION__ `, with double underscores preceding and succeeding, which is conflicting with the markdown syntax. may be better just wrap them around back quotes.

Reply:
got it.